pkgs/applications/networking/cluster/k3s/1_23/default.nix

```diff
@@ -174,11 +175,15 @@ let
     vendorSha256 = k3sVendorSha256;
 
     nativeBuildInputs = [ pkg-config ];
-    buildInputs = [ libseccomp ];
+    buildInputs = [ libseccomp sqlite.dev ];
 
     subPackages = [ "cmd/server" ];
     ldflags = versionldflags;
 
+    preBuild = ''
+      export GOFLAGS="-tags=libsqlite3,linux $GOFLAGS"
```

Comment:
Won't the `linux` thing affect MacOS builds?

Reply:
Any Mac user that can test?

Reply:
As far as I can tell, the package isn't buildable on MacOS since it already depends on `libseccomp` and `util-linux` (both only available for aarch64-linux, i686-linux, x86_64-linux).

Reply:
Makes sense, thanks.